trimmer_decorator.rb

```diff
@@ -0,0 +1,12 @@
+# Defines a trimmer_decorator class
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    name = @nameable.correct_name
+
+    if name.length > 10
+      name[0, 10]
+    else
```

Comment:
Please use `...` instead of a coma to trim the name i.e `name[0...10]`.

Reply:
Implemented